### DISCUSSION TITLE
Join Grub1 with Ventoy

### DISCUSSION THREAD

**Discussion by LEENO82** [ventoy/Ventoy#731] [Q&A]
Hallo
i have pendrive built with grub1

Partition M: is just for save files, is possible to use this partition for add Ventoy? Keeping Grub1 working in partition L:

<img src="https://i.imgur.com/VzwtFWB.jpg">

**Comment by ventoy** [comment]
NO.
Ventoy must reformat the disk.

**Reply by LEENO82** [reply]
after reformated with ventoy, can i add grub1 boot too?

**Reply by chromer030** [reply]
A disk cannot contain 2 boot-loader simultaneously, only one (first) is detectable and usable by BIOS/UEFI at boot-loading.
And using a boot-loader inside another is not recommended. (like using grubfm inside ventoy.)

**Reply by LEENO82** [reply]
ok thanks for answer
so i hope in next version maybe supported placing grub1 content in GRUB1 folder in ventoy disk
